cmd/trusty-tool/main.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/go-phorce/dolly/ctl"
+	"github.com/go-phorce/dolly/xlog"
+	"github.com/go-phorce/trusty/cli"
+	"github.com/go-phorce/trusty/cli/csr"
+	"github.com/go-phorce/trusty/version"
+)
+
+var logger = xlog.NewPackageLogger("github.com/go-phorce/trusty/cmd", "trusty-tool")
+
+const (
+	rcError   = 1
+	rcSuccess = 0
+)
+
+func main() {
+	// Logs are set to os.Stderr, while output to os.Stdout
+	rc := realMain(os.Args, os.Stdout, os.Stderr)
+	os.Exit(int(rc))
+}
+
+func realMain(args []string, out io.Writer, errout io.Writer) ctl.ReturnCode {
+	formatter := xlog.NewColorFormatter(errout, true)
+	xlog.SetFormatter(formatter)
+
+	app := ctl.NewApplication("trusty-tool", "A command-line utility for issuing offline certificates.").
```

Comment:
"offline certificates".. what does it mean? I heard about "offline CAs" but not "offline certificates".